core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -444,7 +445,7 @@ class SparkContext(config: SparkConf) extends Logging {
     _eventLogCodec = {
       val compress = _conf.get(EVENT_LOG_COMPRESS)
       if (compress && isEventLogEnabled) {
-        Some(CompressionCodec.getCodecName(_conf)).map(CompressionCodec.getShortName)
+        Some(EventLogFileWriter.codecName(_conf)).map(CompressionCodec.getShortName)
```

Comment:
Let's not make SparkContext further depend on these classes. Just change the other one to match how this works?

Reply:
Any reason for reducing dependencies on "these classes" ?

"change the other one" ?
Maybe, just use `_conf .get(EVENT_LOG_COMPRESSION_CODEC)` without adding dependency on `EventLogFileWriter` should be OK ?

Reply:
yea `_conf.get(EVENT_LOG_COMPRESSION_CODEC)` should be good enough

Reply:
+1 for @srowen and @cloud-fan 's advice.